src/uct/ib/base/ib_iface.c

```diff
@@ -1031,8 +1032,10 @@ ucs_status_t uct_ib_verbs_create_cq(uct_ib_iface_t *iface, uct_ib_dir_t dir,
                            preferred_cpu);
     }
 
-    if (!cq) {
-        ucs_error("ibv_create_cq(cqe=%d) failed: %m", cq_size);
+    if (cq == NULL) {
+        ucs_snprintf_safe(message, sizeof(message), "ibv_create_cq(cqe=%d)",
+                          cq_size);
+        uct_ib_mem_lock_limit_msg(message, errno);
```

Comment:
I think we should save errno in local var before calling ucs_snprintf_safe, in case it gets overridden